operators/cmd/manager/main.go

```diff
@@ -135,11 +135,10 @@ func init() {
 	)
 	Cmd.Flags().String(
 		DebugHTTPServerListenAddressFlag,
-		":6060",
+		"localhost:6060",
 		"Listen address for debug HTTP server (only available in development mode)",
 	)
 
-	viper.BindPFlags(Cmd.Flags())
```

Comment:
This was done twice, removed the first duplicate (which didn't have error handling)